charts/prometheus-cloudwatch-exporter/templates/NOTES.txt

```diff
@@ -11,5 +11,5 @@
 {{- else if contains "ClusterIP" .Values.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "prometheus-cloudwatch-exporter.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
   echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl port-forward $POD_NAME 8080:80
+  kubectl port-forward $POD_NAME 8080:9106 --namespace {{ .Release.Namespace }}
```

Comment:
port seems to be different: 
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-couchdb-exporter/templates/deployment.yaml#L41

Reply:
Thanks @monotek 

Sorry I think I type the wrong topic. 
change the topic from [prometheus-couchdb-exporter] to [prometheus-cloudwatch-exporter] now.